packages/components/src/components/Progress/ProgressExpandButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Pressable, View } from "react-native";
+import { Typography } from "../Typography";
+import { Icon } from "../Icon";
+import { EDSStyleSheet } from "../../styling";
+import { useStyles } from "../../hooks/useStyles";
+import { ProgressStatus } from "./types";
+
+type ProgressExpandButtonProps = {
+    taskStatus: ProgressStatus;
+    taskCounter: number;
+    isExpanded: boolean;
+    toggleExpand: () => void;
+};
+
+export const ProgressExpandButton = ({
+    taskStatus,
+    taskCounter,
+    isExpanded,
+    toggleExpand,
+}: ProgressExpandButtonProps) => {
+    const styles = useStyles(themeStyles);
+
+    return (
+        <View>
+            {taskStatus !== "notStarted" && taskCounter > 0 && (
+                <Pressable style={[styles.dropDownContainer]} onPress={toggleExpand}>
```

Comment:
Could this be a normal Button (from mad components) instead? We could use a ghost button for example. This way, we get more reuse of functionality and don't have to style or change this should we make changes to the visual expression of our other components in the future.

Also great for having the extra padding that comes with the buttons in regards to touch surface area.

Reply:
Good idea